pint/models/parameter.py

```diff
@@ -44,15 +45,31 @@ class Parameter(object):
 
         print_value is a function that converts the internal value to
           a string for output.
-
+        bare_value is the purely numerical number from value property
+        get_bare_value is the function that gets bare value from value
     """
 
     def __init__(self, name=None, value=None, units=None, description=None,
             uncertainty=None, frozen=True, aliases=None, continuous=True,
-            parse_value=fortran_float, print_value=str):
-        self.value = value
+            parse_value=fortran_float, print_value=str,get_value = None,
+            get_bare_value=lambda x: x):
         self.name = name
         self.units = units
```

Comment:
This could use a comment describing what is meant by each attribute. In particular, it should clearly specify whether self.units is a string or an instance of an astropy.unit